packages/trader/package.json

```diff
@@ -8,6 +8,9 @@
     "serve": "echo \"Serving...\" && sudo webpack-dev-server --config \"./build/webpack.config.js\"",
     "build": "f () { npm run test && webpack --config \"./build/webpack.config.js\" --env.base $1;}; f",
     "test": "mochapack -r babel-polyfill -r jsdom-global/register -r mock-local-storage --webpack-config \"./build/webpack.config-test.js\" \"src/**/__tests__/*.js\"",
+    "test:eslint": "eslint \"./src/**/*.?(js|jsx)\"",
+    "test:stylelint": "stylelint \"./src/sass/**/*.s?(a|c)ss\"",
```

Comment:
```suggestion
    "test:stylelint": "stylelint \"./src/sass/**/*.scss\"",
```

Reply:
We need to check for both, otherwise stylelint might miss a file or 2 when linting.